src/Filters/Block.php

```diff
@@ -712,20 +711,24 @@ private function getAttributesBlock($filter, $selectedFilters, $idLang)
 
         foreach ($results as $key => $values) {
             $idAttribute = $values['id_attribute'];
-            $count = $values['c'];
+            if (!isset($attributes[$idAttribute])) {
+                continue;
+            }
 
+            $count = $values['c'];
             $attribute = $attributes[$idAttribute];
             $idAttributeGroup = $attribute['id_attribute_group'];
             if (!isset($attributesBlock[$idAttributeGroup])) {
                 $attributeGroup = $attributesGroup[$idAttributeGroup];
 
                 $attributeGroupLayeredInfos = $this->getAttributeGroupLayeredInfos($idAttributeGroup, $idLang);
-                if (empty($attributeGroupLayeredInfos)) {
-                    continue;
+                if (!empty($attributeGroupLayeredInfos)) {
+                    list($urlName, $metaTitle) = array_values($attributeGroupLayeredInfos);
+                } else {
+                    $urlName = null;
```

Comment:
```suggestion
                    $urlName = $metaTitle  null;
```
Not sure if useful 😄